demo/web/src/main/layouts/layout-introduction.json

```
@@ -112,7 +112,7 @@
                 "component": "MarkdownNotebookPanel",
                 "props": {
                   "metadata": {
-                    "id": "/00 The Deephaven IDE.md",
+                    "id": "/Examples/00 The Deephaven IDE.md",
                     "itemName": "/00 The Deephaven IDE.md"
```

Need to change this link as well.
```suggestion
                    "itemName": "/Examples/00 The Deephaven IDE.md"
```